incubator/jasperserver/values.yaml

```diff
@@ -3,6 +3,10 @@
 ##
 image: bitnami/jasperserver
 
+## Bitnami image Tag 
+#
+imageTag: 6.3.0-r3
```

Comment:
In our charts we have decided to use a single value for the image and tag named `image`. I suggest you combine the `imageTag` field with `image` so that its `image: bitnami/jasperserver:6.3.0-r3`